notebooks/Network_Usage.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "## Basic Tutorial for `pysal.spaghetti`\n",
     "\n",
-    "#### updated 07/2018"
+    "#### updated 08/2018"
```

Comment:
this is a great idea @jGaboardi! I wonder if there's a way to print this directly from the notebook metadata, like a "last ran on YYYY-MM-DD" thing.

Reply:
@ljwolf thanks. I'll look into setting something up like that.